fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraSearchIT.java

```diff
@@ -435,6 +435,52 @@ public void testMalformedCondition() throws Exception {
         }
     }
 
+    @Test
+    public void testDefaultOrdering() throws Exception {
+        final var prefix = getRandomUniqueId();
+        final var resources = createResources(prefix, 3);
+        final var condition = FEDORA_ID + "=" + prefix + "*";
+        final String searchUrl = getSearchEndpoint() + "condition=" + encode(condition) +
+                "&order_by=fedora_id&order=asc";
```

Comment:
Since this test is named to be testing the "default" ordering, I would suggest removing the `order_by` and `order` query parameters... and rely on the defaults.
A new, separate test could be useful to test `desc` order.